pkgs/servers/nosql/apache-jena/fuseki-test.nix

```
@@ -0,0 +1,18 @@
+{ lib, runCommandNoCC, apache-jena-fuseki, curl }:
```

```suggestion
{ lib, runCommand, apache-jena-fuseki, curl }:
```
This is an alias and breaks eval. Please fix asap!